src/components/Pages/RequestPage/RequestPage.tsx

```diff
@@ -158,37 +167,42 @@ export const RequestPage = () => {
     }
   }, [])
 
-  const onApproveSignInVerification = useCallback(async () => {
-    getAnalytics().track(TrackingEvents.CLICK, {
-      action: ClickEvents.APPROVE_SING_IN
-    })
-    setIsLoading(true)
-    const provider = providerRef.current
-    try {
-      if (!provider) {
-        throw new Error('Provider not created')
+  const onApproveSignInVerification = useCallback(
+    async (skipTrack?: boolean) => {
```

Comment:
There is no need for this param, you can use targetConfig.skipVerifyCode instead